api/v1beta1/types.go

```diff
@@ -0,0 +1,196 @@
+/*
+Copyright 2021 Absa Group Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Generated by GoLic, for more details see: https://github.com/AbsaOSS/golic
+*/
+package v1beta1
+
+import (
+	"k8s.io/api/extensions/v1beta1"
+)
+
+type IngressSpec struct {
```

Comment:
Can we reference the upstream source of `IngressSpec` for future?

Reply:
@ytsarev , good idea, amended